### ISSUE TITLE
visionOS support

### ISSUE THREAD

**Issue by cplr** [yoshisuga/MAME4iOS#455]
The iOS version should work just fine, but what about putting the window in a virtual cab in your room? 😎 

I'm imagining pairing this with the Mayflash F700 (bluetooth arcade controller)...

**Comment by ToddLa** [comment]
Yes this would be a obvious idea, next time we do a release we should at least attach a [UIOrnament](https://developer.apple.com/documentation/swiftui/uiornament) to the main window, or use multiple windows for game play and game selection. 

We need to be careful supporting multiple windows because the MAME core is inherently single instance. 

Getting 3D models of Arcade cabinets, standup and cocktail, also Consoles connected to a TeeVee, with moveable buttons and sticks, is not a trivial task, know any 3D artists? Or know any stock 3D models???
